refactor/data/app/api/api_v0/endpoints/fs.py

```diff
@@ -58,7 +58,7 @@
 
 
 @router.post("/upload", status_code=200, response_model=UploadResponse)
-def upload_file(*, file: UploadFile) -> Any:
+def upload_file(*, file: UploadFile, overwrite: bool = False) -> Any:
```

Comment:
I also think that by default `overwrite` should be `True`.

Reply:
I think the default should be to _not_ delete files. User should have to send an extra param to destroy data.

Reply:
Oh, sorry, you're write. I meant that we need to pass `overwrite=True` when the result service is interacting with data service. So every time result calls this endpoint, overwrite should always be done.